app/models/presenter.rb

```diff
@@ -11,4 +11,8 @@ def show_questions(category)
       Question.order_by_update.first(25)
     end
```

Comment:
I'm uncertain if we're going to have more than one presenter file. Should we maybe rename this more explicitly?